BenchmarkTest00233.py

```
@@ -0,0 +1,9 @@
+#{fact rule=untrusted-deserialization@v1.0 defects=0}
+
+import jsonpickle
+
+def ok():
+    # ok: avoid-jsonpickle
+    obj = jsonpickle.decode('foobar')
```

Ruff (0.13.1)

Local variable `obj` is assigned to but never used

Remove assignment to unused variable `obj`

(F841)